lib/std/monotimes.nim

```diff
@@ -84,11 +84,11 @@ elif defined(windows):
     importc: "QueryPerformanceFrequency", stdcall, dynlib: "kernel32".}
 
 proc getMonoTime*(): MonoTime {.tags: [TimeEffect].} =
-  ## Get the current `MonoTime` timestamp.
+  ## Gets the current `MonoTime` timestamp.
   ##
   ## When compiled with the JS backend and executed in a browser,
-  ## this proc calls `window.performance.now()`, which is not supported by
```

Comment:
The statement is unnecessary

https://caniuse.com/?search=performance.now

95.31% of browsers support `window.performance.now()`.